sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1236,6 +1260,12 @@ export class ContainerClient extends StorageClient {
     }>;
 }
 
+// @public
+export interface ContainerCpkScopeInfo {
```

Comment:
better to name it `ContainerCustomerProvidedKeyScopeInfo`?

Reply:
Cpk seems to be used fairly extensively - given how long "CustomerProvidedKey" is, I'm fine with the `cpk` initialism personally. If the other languages ditch `cpk` however I'd want to align.

Reply:
Since we already shipped `CpkInfo`, it is more important to be consistent with released package than across languages. so `Cpk` it is.

Reply:
ContainerEncryptionScopeInfo is another option (based on the rename of cpkScope to encryptionScope elsewhere).

Reply:
I'm considering just "ContainerEncryptionScope"?

Reply:
+1 to dropping info. Thanks @annatisch!